packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 51.10.0 - April 21, 2021
+
+## Fixed
+
+#### docs
+
+- Update docs to reflect Apex library setting has been removed ([PR #3148](https://github.com/forcedotcom/salesforcedx-vscode/pull/3148))
+
+#### salesforcedx-vscode-apex
+
+- Update "All tests" option for `SFDX: Run Apex Tests` command to match CLI behavior ([PR #3126](https://github.com/forcedotcom/salesforcedx-vscode/pull/3126))
+
+#### salesforcedx-vscode-core
+
+- Fix issues with the following types for deploy/retrieve library ([PR #3147](https://github.com/forcedotcom/salesforcedx-vscode/pull/3147), [Issue #3114](https://github.com/forcedotcom/salesforcedx-vscode/issues/3114), [Issue #3157](https://github.com/forcedotcom/salesforcedx-vscode/issues/3157)):
+  - TimeSheetTemplate
```

Comment:
Would it be worth mentioning that these were also added? timesheettemplate and accountrelationshipsharerule

Reply:
Ah yeah i forgot account relationship, thanks